src/components/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
```

Comment:
* [ ] Kindly remove this code, there are no requirement for styling, at least not for now. unrelated code must be removed.